oaknational/Oak-Web-Application
# Feat/learning themes filter

## Description

add learning them filter to units listing page
updated the units query to include "learning thmese"
refactored the MobileBlogFilter to be a generic mobile filter container which can hold any filter

## Issue(s)

Fixes #1049 

## How to test

1.  go to any units listing page and 
2. Click on any of the learning filters  the data filtered by learning theme
3. You should see the data filtered by learning theme
## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

![image](https://user-images.githubusercontent.com/48293828/214625927-428fb2a4-24e6-4045-9e22-1d4b06fa8a75.png)
![image](https://user-images.githubusercontent.com/48293828/214626049-094afe6d-7e69-46b8-9af8-a8d27d739f6e.png)

## Checklist
